### DISCUSSION TITLE
Add classname in tailwindcss app through api (Tailwindcss Website Builder)

### DISCUSSION THREAD

**Discussion by mickeymaruf** [tailwindlabs/tailwindcss#13065] [Help] [answered]
Hey,

I was working on a website builder app in Next.js/Tailwindcss. I was trying to make something similar to https://www.loopple.com/ their platform where I can drag and drop elements and design them using css classes.

Here is the case:
I have an input field where I can add CSS classes and it will reflect on my website just in time. If I use raw CSS it's reflecting but for Tailwindcss.

So how can I enable this feature? Is there any approach that I can incorporate to achieve this

Thanks :)

**Comment by wongjn** [comment, marked as answer]
You could consider:
- [`safelist`ing the list of classes](https://tailwindcss.com/docs/content-configuration#safelisting-classes), if there is a finite number.
- Use the [Play CDN](https://tailwindcss.com/docs/installation/play-cdn).
- Use `style` attributes instead of Tailwind classes.

**Reply by mickeymaruf** [reply]
Hey thanks! the Play CDN method worked for me.

**Reply by hoangchungk53qx1** [reply]
nice @mickeymaruf  ✌️
